jujugui/static/gui/src/app/init/utils.js

```diff
@@ -379,7 +379,11 @@ utils.switchModel = function(
     });
     return;
   }
-  const switchModel = utils._switchModel.bind(this, modelAPI, model, clearProfileState);
+  const updateModelUUID = uuid => {
```

Comment:
Yeah that's a good point however we also need `state` passed in then which is really just another argument.

I'm hoping that this can be dramatically cleaned up with jujulib (as mentioned above)